src/Command/Overview/HearingOverviewCreateCommand.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+namespace App\Command\Overview;
+
+use App\Command\ArchiverCommand;
+use App\Entity\Archiver;
+use App\Overview\HearingOverviewHelper;
+use App\Repository\ArchiverRepository;
+use Symfony\Component\Console\Attribute\AsCommand;
+use Symfony\Component\Console\Exception\InvalidArgumentException;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Logger\ConsoleLogger;
+
+#[AsCommand(
+    name: 'app:overview:hearing-create',
```

Comment:
Why not create hearing?

Reply:
`app:overview:hearing` is sort of a namespace and `create` is the command … That's my best answer.